openapi/paylocity/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["paylocity", "HR", "payroll"]
+org = "ballerinax"
+name = "payroll"
```

Comment:
Here name must be `paylocity` not `payroll`

Reply:
Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/30ae73f063391071311ed2f7f5a9639bfe05a36c